src/lib/holocene/textarea.svelte

```diff
@@ -60,6 +95,26 @@
   }
 
   .error-msg {
-    @apply border-danger font-primary text-sm font-normal text-danger;
+    @apply break-words border-danger font-primary text-sm font-normal text-danger;
```

Comment:
One suggestion: add `h-auto min-h-[1.5rem]` here to keep a consistent height so there's no jumping when the error message does appear, and on small screens it will grow to give the message and the count more room and not bleed into content below.

Before
<img width="500" alt="Screen Shot 2023-11-20 at 9 17 52 AM" src="https://github.com/temporalio/ui/assets/7967403/2d56c9cd-21d1-4184-b776-a1bc8180e8a0">

After
<img width="500" alt="Screen Shot 2023-11-20 at 9 18 09 AM" src="https://github.com/temporalio/ui/assets/7967403/2f85f396-2bc3-4ec3-8656-25ca4b4a361a">

Reply:
Done ✅